README.rst

```diff
@@ -111,6 +111,10 @@ OPTIONS
 
 -u user     User to run as. Defaults to ``varnish``.
 
+-w curl-timeout
```

Comment:
Maybe "curl-timeout" could be more intuitive. Maybe "push-timeout"? Don't know, and we can always change it later (which in practice never happens).